librarian_packages/server/tests/test_store.py

```diff
@@ -0,0 +1,21 @@
+# -*- mode: python; coding: utf-8 -*-
+# Copyright 2019 the HERA Collaboration
+# Licensed under the 2-clause BSD License
+
+"""Test code in librarian_server/store.py
+
+"""
+
+from __future__ import print_function, division, absolute_import, unicode_literals
```

Comment:
Virtually 0% chance of this mattering here, but while I used to use the `unicode_literals` future, I don't anymore, because when you're writing code that aims for Python 2/3 compat you sometimes need to be able to create literals that are of type `str` regardless of whether that means bytes (Python 2) or unicode (Python 3).

Reply:
Thanks for pointing this out. It won't matter soon because of the py3 changes, but good to know for compatibility.